org.nongnu.gsequencer.gsequencer.json

```diff
@@ -0,0 +1,241 @@
+{
+    "app-id": "org.nongnu.gsequencer.gsequencer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gsequencer",
+    "rename-desktop-file": "gsequencer.desktop",
+    "rename-appdata-file": "gsequencer.appdata.xml",
+    "rename-icon": "gsequencer",
+    "finish-args": [
+	/* X11 + XShm access */
+	"--share=ipc",
+	"--socket=x11",
+	/* Wayland */
+	"--socket=wayland",
+	/* Note playback */
+	"--socket=pulseaudio",
+	/* Soundcard and MIDI */
+	"--device=all",
+	/* Allow loading, saving files from anywhere (portals don’t work yet) */
+	"--filesystem=host"
+    ],
+    "cleanup": [
+	"/include",
+	"/lib/pkgconfig",
+	"/share/man",
+	"/share/doc",
+	"/share/gtk-doc",
+	"*.la"
+    ],
+    "modules": [
+	"gtk2.json",
+	{
+	    "name": "ladspa",
+	    "no-autogen": true,
+	    "subdir": "src",
+	    "make-install-args": [
+		"INSTALL_PLUGINS_DIR=/app/lib/ladspa",
```

Comment:
Broken whitespace throughout this file.

Reply:
I use emacs

Reply:
OK? I'm sure it understands how spaces work. Each list should be indented:


```json

"foo": [
  "bar",
  "baz",
]
```

Reply:
I just did untabify.

Reply:
:+1: